coordinator/src/validator.rs

```diff
@@ -11,11 +11,24 @@ pub type Bytes = Vec<u8>;
 
 pub type VoteWeight = u64;
 
+#[derive(Clone, Debug, PartialEq, Eq)]
 pub struct Event {
     pub key: &'static str,
     pub value: Bytes,
 }
 
+impl Encodable for Event {
```

Comment:
Encoded, Decoded traits are required because I need to store Events to eventDB.
I think it was a mistake to put the change in this commit.
I'll create a separate commit before the commit implementing EventDB.
Please tell me if you think I should squash the commits.

Reply:
I think it's your choice. If you ask me, I'll separate the commits when the commit implementing EventDB is too big, otherwise, I'll squash them into one.

Reply:
I created a separate commit before the commit implementing EventDB.